setup.py

```diff
@@ -34,7 +34,7 @@ def get_long_description() -> str:
     package_dir={"": "src"},
     include_package_data=True,
     zip_safe=False,
-    install_requires=["starlette==0.13.*", "tartiflette>=1.0,<1.4"],
+    install_requires=["starlette==0.16.*", "tartiflette>=1.0,<1.4"],
```

Comment:
I wonder if we ought to use this instead…

```suggestion
    install_requires=["starlette>=0.13,<1.0", "tartiflette>=1.0,<1.4"],
```

I assume (?) nothing changed that broke compatibility with Starlette 0.13+ since then, so we don't _necessarily_ need to enforce anything stricter than that. Might need to review the Starlette changelog to confirm.

WDYT?

Reply:
It looks like we're going to have a failing test with Starlette 0.15+.

Running tests locally makes the 2 variants of `test_if_subscriptions_disabled_then_cannot_connect` in `tests/test_subscription.py` fail:

```python
        with TestClient(app) as client:  # type: TestClient  # type: ignore
            with pytest.raises(WebSocketDisconnect):
>               client.websocket_connect("/subscriptions")
E               Failed: DID NOT RAISE <class 'starlette.websockets.WebSocketDisconnect'>

tests/test_subscription.py:28: Failed
```

Seems to be because:

> TestClient.websocket_connect() now must be used as a context manager.

https://www.starlette.io/release-notes/#0150

So we'd need to account for that here?

Reply:
@florimondmanca - I think relaxing the dependency should be okay. Checking throught the changelog, the only breaking change was the `TestClient.websocket_connect()` as you outlined above.

I've pushed a change for the starlette dependacy as suggested above and I believe my change to the above test case should accounted for.

Locally, I've run the test suite across starlette 0.13, 0.14, 0.15 and 0.16 without any issues.